magento/magento2
# Fixed #23897 Sales Configuration in the Minimum Amount field have not proper validation

### Description (*)

Magento 2.3 Sales configuration in the `Sales` have `Minimum Order Amount` have a field `Minimum Amount` resolved proper validation.

### Fixed Issues (if relevant)

1. #23897 : Sales Configuration in the Minimum Amount field have not proper validation

### Manual testing scenarios (*)

1. Login to the Magento 2 admin.
2. Go to the STORES >> Settings >> Configuration >> Sales >> Sales >> Minimum Order Amount:
3. Minimum Amount: add any text value
4. Save Config
5. There should be allow only digits

### Questions or comments


### Contribution checklist (*)
